dataGenerator.js

```diff
@@ -0,0 +1,94 @@
+const Faker = require('faker');
+
+const getRandomInt = (max) => Math.floor(Math.random() * Math.floor(max));
```

Comment:
not sure if the second math.floor on max is necessary, but probably good practice